config/rmt.yml

```
@@ -3,27 +3,26 @@
 # You can create rmt.local.yml to override settings in this file.
 
 database: &database
-  host: localhost
-  database: rmt_development
-  username: rmt
-  password: rmt
+  host: <%= ENV.fetch('MYSQL_HOST') { 'localhost' } %>
+  database: <%= ENV.fetch('MYSQL_DATABASE') { 'rmt_development' } %>
+  username: <%= ENV.fetch('MYSQL_USER') { 'rmt' } %>
+  password: <%= ENV.fetch('MYSQL_PASSWORD') { 'rmt' } %>
   adapter: mysql2
   encoding: utf8
   timeout: 5000
   pool: 5
 
 database_development:
   <<: *database
-  database: rmt_development
 
 database_test:
   <<: *database
   database: rmt_test
 
 scc:
   host: https://scc.suse.com/connect
-  username:
-  password:
+  username: <% ENV['SCC_USERNAME'] %>
+  password: <% ENV['SCC_PASSWORD'] %>
```

Here, the ERB syntax is missing the equal signs:

```suggestion
  username: <%= ENV['SCC_USERNAME'] %>
  password: <%= ENV['SCC_PASSWORD'] %>
```

I took some time to figure it out, but after changing, it worked.

---

Done!